pkg/executive/executive_endpoint.go

```diff
@@ -295,8 +296,11 @@ func (ee *ExecutiveEndpoint) Handler() http.Handler {
 	r.HandleFunc("/limits/writers/{writerName}", ee.handleWriterLimitsUpdate).Methods("POST")
 	r.HandleFunc("/limits/writers/{writerName}", ee.handleWriterLimitsDelete).Methods("DELETE")
 
+	// destructive routes below
+
 	r.HandleFunc("/clear-rows/families/{familyName}", ee.handleClearFamilyRows).Methods("DELETE")
 	r.HandleFunc("/clear-rows/families/{familyName}/tables/{tableName}", ee.handleClearTableRows).Methods("DELETE")
+	r.HandleFunc("/drop-table/families/{familyName}/tables/{tableName}", ee.handleDropTable).Methods("DELETE")
```

Comment:
I think the RESTful thing would be like DELETE /families/{familyName}/tables/{tableName}